.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
Still not sure how this will work with greenkeeper. I had to implement a new script to get greenkeeper to update the package lock in a different project. Could we remove package locks from this PR or is it required?

Reply:
Removing it makes it a little harder to guarantee exact reproducibility in builds, but for now I think it should be fine. The cache is keyed off of the package lock file, but I should be able to just key it off of package.json.